process/elasticproc/logsevents/logsAndEventsProcessor.go

```diff
@@ -265,22 +260,13 @@ func (lep *logsAndEventsProcessor) prepareLogEvent(dbLog *data.Logs, event *data
 		Topics:         hexEncodeSlice(event.Topics),
 		Order:          event.Order,
 		ShardID:        shardID,
+		TxOrder:        execOrder,
+		OriginalTxHash: dbLog.OriginalTxHash,
+		Timestamp:      dbLog.Timestamp,
+		ID:             fmt.Sprintf("%s-%d-%d", dbLog.ID, shardID, event.Order),
```

Comment:
optional: `%s-%d-%d` can be extracted to a constant, `idFormat`

Reply:
done